v0/destinations/singular/config.js

```diff
@@ -0,0 +1,91 @@
+const { getMappingConfig } = require("../../util");
+
+const BASE_URL = "https://s2s.singular.net/api/v1";
+
+/* Supported events in Singular: SessionNotification, EventNotification
+   ref: https://support.singular.net/hc/en-us/articles/360048588672-Server-to-Server-S2S-API-Endpoint-Reference
+ */
+const CONFIG_CATEGORIES = {
+  SESSION_ANDROID: {
+    name: "SINGULARAndroidSessionConfig",
+    type: "track"
+  },
+  SESSION_IOS: {
+    name: "SINGULARIosSessionConfig",
+    type: "track"
+  },
+  EVENT_ANDROID: {
+    name: "SINGULARAndroidEventConfig",
+    type: "track"
+  },
+  EVENT_IOS: {
+    name: "SINGULARIosEventConfig",
+    type: "track"
+  },
+  PRODUCT_PROPERTY: {
+    name: "SINGULAREventProductConfig"
+  }
+};
+
+const SINGULAR_SESSION_ANDROID_EXCLUSION = [
+  "install_source",
+  "asid",
+  "openuri",
```

Comment:
Same here `openuri` will not be available in `properties` its `properties.url` we should put `url` in exclusion list

Reply:
done